homeassistant/components/wemo/fan.py

```diff
@@ -86,6 +86,7 @@ class WemoHumidifier(WemoBinaryStateEntity, FanEntity):
 
     _attr_supported_features = FanEntityFeature.SET_SPEED
     wemo: Humidifier
+    _last_fan_on_mode: FanMode
```

Comment:
was this intentional?

Reply:
Some what 😅 Though it was required for this release but turns out it's actually the next one as that improves the `Enum` type inference.

I could remove it here but since it isn't wrong might as well leave it in already.